src/node_url.cc

```diff
@@ -624,6 +624,13 @@ namespace url {
     }
   }
 
+  static inline void ShortenUrlPath(struct url_data* url) {
```

Comment:
Actually it cannot be a const because the url is modified in the function. Non-const references are not allowed by the linter, so I'll keep the pointer.